### DISCUSSION TITLE
Retrieving SkyBlock Calendar data?

### DISCUSSION THREAD

**Discussion by alefnull** [HypixelDev/PublicAPI#609] [Q&A] [answered]
hi all, i'm honestly not sure if this is the right place to ask, but i don't really know where else to look right now.

i started playing SkyBlock about a month ago and recently decided i wanted to look into building an android app to keep track of the current SkyBlock calendar (if possible) even when i'm not logged in. i've seen one or two web-based projects that do this, but i'm hoping i can do something on android so i can potentially set up mobile notifications of specific events i want to keep track of.

i've looked over all the public API info i could find so far but haven't seen anything about getting the SkyBlock calendar info. is that something that's not available via the API? and if so, does anyone have any idea where i could look to understand how to go about doing this?

again, sorry if this is not the correct place to be asking this.

**Comment by DuckySoLucky** [comment, marked as answer]
Heya, SkyBlock calendar is not available via the API. However it's still possible by creating your own calculator, there are some open source calculators like [SlothPixel's Event Calendar Builder](https://github.com/slothpixel/core/blob/master/store/buildSkyblockCalendar.js). Note: SlothPixel's calculator is not really up to date and it's missing some events like Shen's auction and Mayor events (Marina's Fishing festival, Cole's Mining fiesta etc...)

**Reply by alefnull** [reply]
got it, thanks for the info 😃
